README.md

```diff
@@ -27,4 +27,19 @@ to run `site-scrapers/MAImmunizations.js`
 1. When you're ready to deploy via AWS Lambda, run `npm run predeploy` which will generate `lambda.zip` for you. This needs to stay under 50 MB for you to upload it manually.
 1. Your production environment needs to have the environment variables `AWSS3BUCKETNAME`, `AWSACCESSKEYID`, and `AWSSECRETACCESSKEY` so that it can publish to S3. If you are inserting your own scrapers, set `PROPRIETARY_SITE_SCRAPERS_PATH` in production as well. If you have any scrapers that need to solve reCAPTCHAs, you will also need a `RECAPTCHATOKEN` from the 2captcha service.
 
+## Adding a scraper
+To scrape data from a site, you either need to:
+- figure out which API calls are made (Chrome devtools > Network tab) and make these calls yourself to fetch availability. This is preferable since the underlying APIs are less likely to change than the HTML on a given page.
+- interact with the site (clicking buttons, inspecting HTML elements, etc) using puppeteer
+
+It's often the case that at the time you're trying to write a scraper, there is no availability to scrape. This makes it hard to know what to do in the case of availability.
+To start, you can just return `hasAvailability = true` or `false` and not report specific available appointments.
+
+Then, in the case where you expect to have availability, you can add code to write data (HTML, screenshots, .har files) to S3 for later inspection, or you can even send a Slack notification so that you know to check the site soon! There are examples of these throughout our codebase (search for `slack` or `s3.savePageContent`)
```

Comment:
I think the idea (my conception, anyway!) is to be clear about which examples to look at, because we have some things in our codebase that really should not be copied, and others that are good models. So can we be explicit about giving an example scraper that does slack and/or screenshot saving particularly well?

Reply:
I think the Slack/S3 writing is fine everywhere... I didn't link to specific examples b/c I expect writing to Slack/S3 to eventually be removed once we find availability to write.